BrowserKit/Sources/MenuKit/MenuSquaresViewContentCell.swift

```diff
@@ -50,7 +52,9 @@ final class MenuSquaresViewContentCell: UITableViewCell, ReusableCell, ThemeAppl
         return nil
     }
 
-    private func setupUI() {
+    private func setupUI(mainMenuHelper: MainMenuHelper = MainMenuHelper()) {
```

Comment:
Same as for `HeaderBanner`.

Reply:
This cell still has no way to pass in the `MainMenuHelper` as all the methods are private. Please have a look at how done in `HeaderBanner`.